templates/test/ci/prow-ci-version/patches/kubeadm-bootstrap-windows.yaml

```diff
@@ -2,6 +2,8 @@
   path: /spec/template/spec/files/-
```

Comment:
unrelated to this PR but would folks like to update this repo so the generated templates show up as generated files in PRs?
I can open up a PR to do so if people want. 
@CecileRobertMichon @jsturtevant